### PR TITLE
Updates on Postman collection

### PR DESCRIPTION
**Pull request by RajRele**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -161,11 +161,31 @@ <h2>Step 3: Run Examples</h2>
                 <p>
                     <h5>00 - Get Authenticated (Set Account ID and BaseUrl)</h5>
                     <p>
-                        The example folder <span class="badgeLabel">00 Authentication</span> contains a single method for finding and storing your <code>accountId</code>, obfuscating your password, and setting your <a href="https://docs.docusign.com/esign/guide/usage/quickstart.html#authenticate-and-get-your-base-url" target="_blank">BaseUrl</a> for subsequent API calls.
```
Comment (AaronWDS) [inline index.html]: could have sworn I just sent a commit on my branch that changes this sentence to say Authentication folder as there is no `00` folder.

File: index.html [hunk 2]
```diff
@@ -161,11 +161,31 @@ <h2>Step 3: Run Examples</h2>
...
                     </p>
+
+                        <strong>For Authorization Code Grant:</strong>
+                    <p>
+                        For an Authorization Code Grant Token, begin by opening a web browser and navigating to 
+                        <code>https://account-d.docusign.com/oauth/auth?response_type=code&scope=signature&client_id={{iKey}&redirect_uri=https://google.com</code> replacing {{iKey}} with your integration key and {{callback}} with a callback url (setup when generating the integration key) to obtain consent. After clicking OK, your browser will redirect to that link specified with a long string in the url returned for the <code>code</code> parameter. Use this use the Authoriztion Code for 01 Get Access Token to generate an Access token and a refresh token. Finally, use 02 Get login information to retreive the Account ID and Base Url.
```
Comment (AaronWDS) [inline index.html]: in addition to changing the folder structure I also changed the double handle bars on this link next to ikey as it's missing one on the right side

File: index.html [hunk 3]
```diff
@@ -161,11 +161,31 @@ <h2>Step 3: Run Examples</h2>
...
+                    </p>
+
+
+                    <strong>For JSON Web Token Grant:</strong>
                     <p>
-                        Notice that this (and every other DocuSign API call in our examples) contains an <code>X-DocuSign-Authentication</code> header that includes your <code>Integrator Key</code> (also called a <code>clientId</code>), <code>username</code>, and <code>password</code>. 
+                        For a JSON Web Token (JWT), begin by opening a web browser and navigating to 
+                        <code>https://account-d.docusign.com/oauth/auth?response_type=code&scope=signature%20impersonation&client_id={{iKey}&redirect_uri=https://google.com</code> replacing {{iKey}} with your integration key and {{callback}} with a callback url (setup when generating the integration key) to obtain consent. After clicking OK (or if you have already obtained consent before on this Integration Key).
```
Comment (AaronWDS) [inline index.html]: same issue with missing bracket

**Comment by RajRele** [comment]
I will create a new one

**Pull request closed without merging**